README.md

```diff
@@ -681,6 +681,30 @@ $ lerna exec --scope my-component -- ls -la
 $ lerna run --scope toolbar-* test
 ```
 
+#### --since [ref]
+
+When executing a script or command, scope the operation to packages that have been updated since the specified `ref`. If `ref` is not specified, it defaults to the latest tag.
+
+List the contents of packages that have changed since the latest tag:
+
+```sh
+$ lerna exec --since -- ls -la
```

Comment:
My only complaint with this usage is that it isn't as readable as `lerna exec --only-updated`. The nature of the beast, I suppose. (Aliases that are distinct long options are a bit awkward, and probably better to omit entirely).